modules/swagger-codegen/src/main/java/io/swagger/codegen/CliOption.java

```diff
@@ -20,4 +33,49 @@ public String getDescription() {
     public void setDescription(String description) {
         this.description = description;
     }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public String getDefault() {
+        return defaultValue;
+    }
+
+    public void setDefault(String defaultValue) {
+        this.defaultValue = defaultValue;
+    }
+
+    public CliOption addEnum(String value, String description) {
+        if (this.enumValues == null) {
+            this.enumValues = new LinkedHashMap<String, String>();
+        }
+        if (!enumValues.containsKey(value)) {
+            enumValues.put(value, description);
+        }
+        return this;
+    }
+
+    public Map<String, String> getEnum() {
+        return enumValues;
+    }
+
+    public void setEnum(Map<String, String> enumValues) {
+        this.enumValues = enumValues;
+        this.type = "enum";
+    }
+
+    public String getOptionHelp() {
```

Comment:
Please mark as `@JsonIgnore` to avoid values in the payload like this:

```json
  "apiPackage": {
    "opt": "apiPackage",
    "description": "package for generated api classes",
    "type": "string",
    "optionHelp": "package for generated api classes"
  }
```

We are using `description` as the description for the argument, not `optionHelp`